inlong-sort/sort-formats/format-rowdata/format-rowdata-base/src/main/java/org/apache/inlong/sort/formats/base/DefaultDeserializationSchema.java

```diff
@@ -49,13 +52,25 @@ public abstract class DefaultDeserializationSchema<T> implements Deserialization
      */
     private boolean errorOccurred = false;
 
+    protected FailureHandler failureHandler;
+
     /**
      * The format metric group.
      */
     protected transient FormatMetricGroup formatMetricGroup;
 
     public DefaultDeserializationSchema(boolean ignoreErrors) {
         this.ignoreErrors = ignoreErrors;
+        if (ignoreErrors) {
+            failureHandler = new IgnoreFailureHandler();
+        } else {
+            failureHandler = new NoOpFailureHandler();
+        }
+    }
+
+    public DefaultDeserializationSchema(FailureHandler failureHandler) {
+        this.failureHandler = failureHandler;
+        this.ignoreErrors = false;
```

Comment:
why         this.ignoreErrors = false is alway false? maybe user will pass IgnoreFailureHandler here

Reply:
done